docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,100 @@ sidebar_position: 2
 
 # Quick Start
 
-In this section we are providing a sample application that one can deploy and just use to play around.
+In this quik start, you will see a sample application that you can deploy and
+use to play around.  As part of this exercise, a backing service and an
+application is required.  The backing service is a PostgreSQL database and
+application is a [Spring Boot REST API server][petclinic].
 
-## K8S
+## Creating a PostgreSQL Cluster using a Custom Resource
 
-_Here we provided a small sample on K8s_
+You can install [Crunchy PostgreSQL from OperatorHub.io][crunchy].  After the
+installation, to create a PostgreSQL cluster, download this shell script and run
+it:
 
-## OpenShift
+```
+bash <(curl -s https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/5e719c22b5b381f26841659acc62cda81b6fa351/pgcluster.sh)
+```
 
-_Here we provided a small sample on OpenShift_
\ No newline at end of file
+Ensure all the pods in `my-postgresql` is running (it will take few minutes):
+
+```
+kubectl get pod -n my-postgresql
+```
+
+You should see output something like this:
+
+```
+NAME                                          READY   STATUS    RESTARTS   AGE
+backrest-backup-hippo-9gtqf                   1/1     Running   0          13s
+hippo-597dd64d66-4ztww                        1/1     Running   0          3m33s
+hippo-backrest-shared-repo-66ddc6cf77-sjgqp   1/1     Running   0          4m27s
+```
+
+Now you can initialize the database:
+
+```
+bash <(curl -s https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/603e288541dbd1d55596ca1b520d7f2a4f1ce76b/init-database.sh)
+```
+
+## Application Deployment
+
+Now you can deploy the `spring-petclinic-rest` app with this `Deployment`
+configuration:
+
+```
+bash <(curl -s https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/603e288541dbd1d55596ca1b520d7f2a4f1ce76b/app-deployment.sh)
+```
+
+Now you cab create the ServiceBinding custom resource:
+
+```
+cat <<-EOF > "spring-petclinic-rest-binding.yaml"
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+    name: spring-petclinic-rest
+	namespace: my-postgresql
+spec:
+    services:
+    - group: "crunchydata.com"
+      version: v1
+      kind: Pgcluster
+      name: hippo
+    - group: ""
+      version: v1
+      kind: Secret
+      name: hippo-hippo-secret
+    application:
+      name: spring-petclinic-rest
+      group: apps
+      version: v1
+      resource: deployments
+    mappings:
+    - name: type
+      value: "postgresql"
+EOF
+
+kubectl apply -f "spring-petclinic-rest-binding.yaml"
+```
+
+There are two service resources listed, the first one is the `Pgcluster` custom
+resource and the second one is the Secret resource generated by the backing
+service.
```

Comment:
IMHO this details is not really needed now - I would remove it and explain in some later Sections.

Reply:
Removed. 884c5bfb1fced4a83b5ccfec68de05a84fa49ec1